tests/contracts/nonfungible-token/contract.scilla

```diff
@@ -0,0 +1,411 @@
+(***************************************************)
+(*               Associated library                *)
+(***************************************************)
+import BoolUtils
+
+library NonfungibleToken
+
+let one_msg = 
+  fun (msg : Message) => 
+  let nil_msg = Nil {Message} in
+Cons {Message} msg nil_msg
+
+(* Checks and see if an address is a contract owner *)
+let checkContractOwner =
```

Comment:
This can be simplified to
```
let checkContractOwner =
￼     fun (msgSender: Address) => 
￼     fun (contractOwner: Address) =>
￼         builtin eq msgSender contractOwner
```

Reply:
Fixed in 14ef803976f2d20b5753993b7c2ba4bf331e1dbb